packages/flutter/test/widgets/platform_view_test.dart

```diff
@@ -562,7 +563,12 @@ void main() {
             height: 100.0,
             child: AndroidView(
               viewType: 'webview',
-              gestureRecognizers: <OneSequenceGestureRecognizer> [new VerticalDragGestureRecognizer()],
+              gestureRecognizers: <Factory<OneSequenceGestureRecognizer>> [
+                new Factory<OneSequenceGestureRecognizer>(
+                  () => new VerticalDragGestureRecognizer(),
+                  type: VerticalDragGestureRecognizer,
```

Comment:
if we could make the type argument be VerticalDragGestureRecognizer, then we wouldn't need to specify the type here.

Reply:
done